### DISCUSSION TITLE
Add support for jiff as datetime crate

### DISCUSSION THREAD

**Discussion by MultisampledNight** [SeaQL/sea-orm#2408] [Ideas]
[jiff](https://crates.io/crates/jiff) is an absolute pleasure to use. It'd be amazing if it was possible to use `sea generate entity --date-time-crate jiff` to instruct SeaORM to use its data types instead.

**Comment by mirsella** [comment]
would be really nice, jiff is highly considered in the rust community

**Comment by AuroraTea** [comment]
All for it! May need an issue!

**Comment by Huliiiiii** [comment]
Converted to issue #2896
